### ISSUE TITLE
disable rbd plugin on one node

### ISSUE THREAD

**Issue by manju-rn** [rook/rook#13933]
I have 3 node microk8s cluster - all are LXD based, 2 are VM and 1 is a container - All are Ubuntu 23.10
The 2 VMs work fine and `csi-rbdplugin-xxxx` works fine
The 1 LXC container has CrashLoopback for rbd pods and the error is:
`csi-rbdplugin-vbql8                             0/2     CrashLoopBackOff` 
Error: `modprobe args: [rbd]): "modprobe: FATAL: Module rbd not found in directory /lib/modules/6.2.0-39-generic\n"`

The LXC container does not have rbd module and i am okay with that since the 2 VM nodes work fine. 
**How can i disable the running of `csi-rbdplugin` in the LXC container. Which option I should choose in operator.yaml to allow only running in VM nodes?**

**Comment by manju-rn** [comment]
> @manju-rn you can use taints/tolerations to customize the rbdplugin
> 
> https://github.com/rook/rook/blob/c409a6ae468bc9db6ad2eee1dd15e4dac9a7c03a/deploy/examples/operator.yaml#L204-L214

Thanks.  help me understand, this will stop even the FS and Objectorage plugin also?  I want only the RBD plugin to stop, since Objectstorage works on LXC container

**Comment by Madhu-1** [comment]
in that case you can use https://github.com/rook/rook/blob/c409a6ae468bc9db6ad2eee1dd15e4dac9a7c03a/deploy/examples/operator.yaml#L228-L231 its specific to RBD plugin.

**Comment by manju-rn** [comment]
Thanks.  So just for my understanding
This is for both CephFS and RBD plugin
> Lines 204 to 214 in [c409a6a](https://github.com/rook/rook/commit/c409a6ae468bc9db6ad2eee1dd15e4dac9a7c03a)

This is for only RBD Plugin
> Lines 228 to 231 in [c409a6a](https://github.com/rook/rook/commit/c409a6ae468bc9db6ad2eee1dd15e4dac9a7c03a)

And this is for FS
> https://github.com/rook/rook/blob/c409a6ae468bc9db6ad2eee1dd15e4dac9a7c03a/deploy/examples/operator.yaml#L245

and so on.

**Comment by Madhu-1** [comment]
Yes correct, Closing this one, Feel free to reopen if it didn't work
